roles/trouter/tasks/main.yml

```diff
@@ -60,21 +55,18 @@
       notify:
         - Restart-trouter
 
-    - name: Add paths to setenv file
-      ansible.builtin.blockinfile:
-        path: "{{ config_folder }}/setenv.sh"
+    - name: Create log file with correct permissions
+      ansible.builtin.file:
+        path: "{{ logs_folder }}/ats-atr.log"
+        state: touch
+        mode: u=rwx,g=rw,o-rwx
         owner: "{{ username }}"
         group: "{{ group_name }}"
-        marker: "# {mark} ATS COMMON ENV VARS"
-        block: |
-          export ATS_HOME={{ binaries_folder }}/transform-service
-          export ATS_TENGINE_AIO_HOST={{ ats_tengine_aio_host }}
-          export ATS_SHARED_FS_HOST={{ sfs_host }}
-        insertafter: EOF
+      changed_when: false
```

Comment:
try with https://github.com/ansible/ansible/issues/30226#issuecomment-433643224

so we still get changed when the file is created but should not yield changed on the next runs